### PR TITLE
Feature/vlto shared preferences

### PR DESCRIPTION
**Pull request by vtomic85**

### CODE REVIEW

File: lib/data/todo_items_repository.dart [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import 'dart:async';
+
+import 'package:rxdart/rxdart.dart';
+import 'package:todo_tracker/data/model/todo_item.dart';
+import 'package:todo_tracker/data/sembast_store.dart';
+
+class TodoItemsRepository {
+  final SembastStore _store = new SembastStore(name: 'todo');
+
+  final _behaviorSubject = BehaviorSubject<List<TodoItem>>();
```
Comment (dusan-milosevic-the-real-one) [inline lib/data/todo_items_repository.dart:10]: perhaps this should also be disposed of... not sure

**Review by dusan-milosevic-the-real-one — APPROVED** [review]

File: lib/app/home/home_view_model.dart [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+class HomeViewModel extends ChangeNotifier {
+  final todoRepository = TodoItemsRepository();
+
+  List<TodoItem> todoItems = [];
+
+  final TextEditingController newTodoController;
+
+  HomeViewModel() : newTodoController = TextEditingController() {
+    newTodoController.addListener(() {
```
[the remark this answers is not in the document]
Comment (vtomic85) [reply lib/app/home/home_view_model.dart:13]: Done (hopefully :))

File: lib/app/home/home_screen.dart [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            Text('My TODOs'),
+            Row(
+              children: [
+                Consumer<ThemeViewModel>(
+                  builder: (context, themeManager, _) => IconButton(
+                    icon: Icon(Icons.nightlight_round),
+                    onPressed: () {
+                      themeManager.getTheme() == themeManager.lightTheme
```
[the remark this answers is not in the document]
Comment (vtomic85) [reply lib/app/home/home_screen.dart]: Done

File: lib/data/todo_items_repository.dart (hunk 1 above)
Comment (vtomic85) [reply lib/data/todo_items_repository.dart:10]: Not sure how... We didn't extend any class so there is no dispose() method to be override. Can we write our own "destructor"?

**Pull request merged**